docs/chain-ids.md

```diff
@@ -0,0 +1,69 @@
+# Chain IDs
+
+| Status | Author                    | Created    | Last updated | License   |
+| ------ | ------------------------- | ---------- | ------------ | --------- |
+| Draft  | The document contributors | 2019-05-23 | 2019-05-28   | CC-BY-4.0 |
+
+## Abstract
+
+In the IOV ecosystem, many tools can be used with different blockchains.
+Client-side, this is every application built on top of IOV-Core. Backend-side,
+this is the Blockchain Name System (BNS), a lookup service that allows assigning
```

Comment:
"which enables the assignment of native blockchain addresses to human readable names"